azurerm/resource_arm_relay_hybrid_connection_test.go

```diff
@@ -0,0 +1,258 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/features"
+)
+
+func TestAccAzureRMHybridConnection_basic(t *testing.T) {
```

Comment:
We should add Relay to these tests names to better categorize them at a glance
```suggestion
func TestAccAzureRMRelayHybridConnection_basic(t *testing.T) {
```